locksmith/src/app.js

```diff
@@ -0,0 +1,66 @@
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const express = require('express')
+const tokenMiddleware = require('./token_middleware')
+const Lock = require('./sequelize')
+
+const app = express()
+const router = express.Router()
+
+router
+  .route('/lock')
+  .put(function(req, res) {
+    let newAddress = req.body.address
```

Comment:
I think a more restful approach would be to add `put` on `/lock/:lockAddress` rather than pass an `address` in the body?